src/main/java/cashleh/transaction/Transaction.java

```diff
@@ -0,0 +1,30 @@
+package cashleh.transaction;
+
+public abstract class Transaction {
+
+    private String description;
+    private double amount;
+
+    public Transaction(String description, double amount) {
```

Comment:
Maybe in v2.0 we could also add currency, dates, type (enumeration containing e.g. salary, interest etc.) and whether the transaction is recurring or not